roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -46,14 +46,16 @@ spec:
       priorityClassName: '{{ control_plane_priority_class }}'
 {% endif %}
       initContainers:
-{% if bundle_ca_crt or projects_persistence|bool or init_container_extra_commands %}
         - name: init
           image: '{{ _init_container_image }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           command:
             - /bin/sh
             - -c
             - |
+              hostname=$MY_POD_NAME
+              receptor --cert-makereq bits=2048 commonname=$hostname dnsname=$hostname nodeid=$hostname outreq=/etc/receptor/tls/receptor.req outkey=/etc/receptor/tls/receptor.key
```

Comment:
why not use `${MY_POD_NAME}` directly?

Reply:
m, does always /etc/receptor/tls exists? another thing for @rooftopcellist to check

Reply:
should we do `bits` something customizable? or go with 4096?

Reply:
I think passing MY_POD_NAME as an env var based on the value of metadata.name should work just fine.  I'm not entirely sure if you can use metadata.name directly there.  I'm Ok with either approach.

Reply:
/etc/receptor/tls should always exist since https://github.com/ansible/awx-operator/pull/1012/files#diff-29de64e6284e07a8a16b351b132d812618c7f4e3485cec09987c2374257b94fcR76-R80

puts it there 

older awx image should be fine since the operator is generating the CA cert/key and the init container is what's generating the client cert/key